src/tests/ftest/cart/corpc/cart_corpc_five_node.yaml

```diff
@@ -24,7 +24,7 @@ hosts: !mux
 tests: !mux
   corpc_prefwd:
     name: corpc_prefwd
-    srv_bin: ../bin/crt_launch
-    srv_arg: "-e tests/test_corpc_prefwd"
-    srv_env: ""
-    srv_ppn: "5"
+    test_servers_bin: ../../../../bin/crt_launch
+    test_servers_arg: "-e ../tests/test_corpc_prefwd"
```

Comment:
can we add cart tests path to PATH in launch.py so we can get rid of relative paths?